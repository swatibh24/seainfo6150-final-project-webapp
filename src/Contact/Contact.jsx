import React from 'react';
import styles from './Contact.module.css';

const Contact = () => (
  <section className={styles.contactContainer}>
        <div className={styles.Contact_part}>
            <h2 className={styles.title}>Contact Us</h2>
            <h3 className={styles.text}>Need Help? Get it now.</h3>
            <div className={styles.method}>
            <div>
              <img  className={styles.logo} src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRSSNwCOUh9OiIFJHTPGAaQcDzMJlS065XtEg&usqp=CAU"
              alt="logo" />
              </div>
              <h2 className={styles.subtitle}>By Phone</h2>
              <div className={styles.desc}>
                <small>(Monday to Friday, 9am to 6pm PST)</small><br/>
                North America Toll-Free:<br/>
                1-111-111-1000<br/>
                International:<br/>
                1-222-000-2222<br/>
              </div>
            </div>
            <div className={styles.method}>
            <div>
              <img
              className={styles.logo}
              src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSlX_7Iy5v4hGWd5RqxrxxBkTAwurwK9pC98A&usqp=CAU"
              alt="logo"
            />
              </div>
              <h2 className={styles.subtitle}>Live Chat</h2>
              <div className={styles.desc}>
                Chat with a member of our in-house team.<br/>
                We are online 24/7<br/>
                Monday to Friday, 9am to 6pm PST<br/>
               </div>
            </div>
            <div className={styles.method}>
            <div>
              <img
              className={styles.logo}
              src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRWMEny7-sAxtOq60-jkos31mmaBoSl9mmlZQ&usqp=CAU"
              alt="logo"/>
              </div>
              <h2 className={styles.subtitle}>By Email</h2>
              <div className={styles.desc}>
                Send us email at myhomerecipes@northeastern.com<br/>
              </div>
            </div>
        
        </div>
    </section>
);

export default Contact;