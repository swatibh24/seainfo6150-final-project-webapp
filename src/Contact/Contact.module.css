.Contact_part p {
    padding: 1px;
    margin-left: 20%;
    background-color: #F5F5F5;
    font-size:20px;
    border:1px,solid,#F5F5F5;
    width:60%;
    height:auto;
    text-align: left;

  }
.contactContainer {
    max-width: 60rem;
    min-width: 20em;
    margin: auto;
    padding: 1px;
    background-color:#F5F5F5;
    font-size:20px;
    width:80%;
    height:auto;
  }

  .method{
    padding: 1px;
    margin-left: 20%;
    font-size:20px;
    width:60%;
    height:auto;
    text-align: left;
    margin-bottom:20px;
  }

  .desc{
    color:rgba(0,0,0,.6);
    text-align:center;
  }


  .title {
    font-weight: bold;
    font-family: Tahoma, Geneva, sans-serif;
    text-align:center;
  }
   .text{
      font-family: Tahoma, Geneva, sans-serif;
      text-align:center;
      color:#5d80cc;
      font-size:20px;
    }

  .subtitle{
      font-size: 20px;
      font-weight: 700;
      line-height: 1.5em;
      text-transform: uppercase;
      letter-spacing: .2em;
      max-width: 275px;
      margin-left: auto;
      margin-right: auto;
      margin-bottom: 1rem;
      text-align: center;
    }

     .logo{
        width: 20%;
        border-radius: 50%;
        object-fit: cover;
        display:block;
        margin:0 auto;
      }


  
  

