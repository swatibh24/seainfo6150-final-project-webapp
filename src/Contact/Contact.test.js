import React from "react";
import { render } from "@testing-library/react";
import Contact from "./Contact.jsx";


describe("Contact tests", () => {
    it("renders correctly", () => {
      const { container } = render(<Contact />);
      expect(container).toMatchSnapshot();
    });
  });