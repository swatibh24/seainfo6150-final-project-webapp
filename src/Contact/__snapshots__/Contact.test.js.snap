// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact tests renders correctly 1`] = `
<div>
  <section
    class="contactContainer"
  >
    <div
      class="Contact_part"
    >
      <h2
        class="title"
      >
        Contact Us
      </h2>
      <h3
        class="text"
      >
        Need Help? Get it now.
      </h3>
      <div
        class="method"
      >
        <div>
          <img
            alt="logo"
            class="logo"
            src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRSSNwCOUh9OiIFJHTPGAaQcDzMJlS065XtEg&usqp=CAU"
          />
        </div>
        <h2
          class="subtitle"
        >
          By Phone
        </h2>
        <div
          class="desc"
        >
          <small>
            (Monday to Friday, 9am to 6pm PST)
          </small>
          <br />
          North America Toll-Free:
          <br />
          1-111-111-1000
          <br />
          International:
          <br />
          1-222-000-2222
          <br />
        </div>
      </div>
      <div
        class="method"
      >
        <div>
          <img
            alt="logo"
            class="logo"
            src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSlX_7Iy5v4hGWd5RqxrxxBkTAwurwK9pC98A&usqp=CAU"
          />
        </div>
        <h2
          class="subtitle"
        >
          Live Chat
        </h2>
        <div
          class="desc"
        >
          Chat with a member of our in-house team.
          <br />
          We are online 24/7
          <br />
          Monday to Friday, 9am to 6pm PST
          <br />
        </div>
      </div>
      <div
        class="method"
      >
        <div>
          <img
            alt="logo"
            class="logo"
            src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRWMEny7-sAxtOq60-jkos31mmaBoSl9mmlZQ&usqp=CAU"
          />
        </div>
        <h2
          class="subtitle"
        >
          By Email
        </h2>
        <div
          class="desc"
        >
          Send us email at myhomerecipes@northeastern.com
          <br />
        </div>
      </div>
    </div>
  </section>
</div>
`;
